### DISCUSSION TITLE
UTM MAC OS WINDOWS 11 SOUND ISSUES

### DISCUSSION THREAD

**Discussion by AggravatedMan** [utmapp/UTM#6175] [macOS]
Hello this is my first post. I was an IT professional over 20 years ago. I am pretty rusty and trying to challenge myself.  I hope I am posting this in the correct location. I installed UTM on my Mac Mini M2 Pro. I have 2 Studio Displays connected to it. I installed Windows 11 pro. Most things seem to be working pretty good. I see in device manager that it looks like its is not installing the drivers for the thunderbolt ports USB-C. The issue I am writing about is that the sound output is terrible quality. It's glitchy, choppy etc. I tried the Mac mini built in speaker as well as an old speaker set using the mini headphone jack. I tried different settings in Windows but it made no difference. Does anyone now how to clean up the sound output so it is clean. I do not need anything more than basic clean sound. Thank you.

**Comment by mcgheavy** [comment]
I having similar issues, except I cant even get the download on my mac.


<img width="437" alt="Screen Shot 2024-03-04 at 12 20 14 PM" src="https://github.com/utmapp/UTM/assets/162179207/98279460-33fb-42ea-add7-1a12b151578a">

**Comment by AggravatedMan** [comment]
that is a totally different issue than I  am having.  Are you running Catalina? I am running Sonoma
